### ISSUE TITLE
e2e flake: Should scale cluster size based on cpu reservation

### ISSUE THREAD

**Issue by mwielgus** [kubernetes/kubernetes#16904]
```
Failed tests:
3 tests failed.
FAILED:  Kubernetes e2e suite.Autoscaling [Skipped][Autoscaling Suite] should scale cluster size based on cpu reservation

Error Message:
null

Stack Trace:
/go/src/k8s.io/kubernetes/_output/dockerized/go/src/k8s.io/kubernetes/test/e2e/cluster_size_autoscaling.go:82
Expected error:
    <*errors.errorString | 0xc20833dc70>: {
        s: "timeout waiting 20m0s for cluster size to be 4",
    }
    timeout waiting 20m0s for cluster size to be 4
not to have occurred

FAILED:  Kubernetes e2e suite.Autoscaling [Skipped][Autoscaling Suite] should scale cluster size based on cpu utilization

Error Message:
null

Stack Trace:
/go/src/k8s.io/kubernetes/_output/dockerized/go/src/k8s.io/kubernetes/test/e2e/cluster_size_autoscaling.go:72
Expected error:
    <*errors.errorString | 0xc2085b6040>: {
        s: "timeout waiting 20m0s for cluster size to be 4",
    }
    timeout waiting 20m0s for cluster size to be 4
not to have occurred

FAILED:  Kubernetes e2e suite.Autoscaling [Skipped][Autoscaling Suite] should scale cluster size based on memory utilization

Error Message:
null

Stack Trace:
/go/src/k8s.io/kubernetes/_output/dockerized/go/src/k8s.io/kubernetes/test/e2e/cluster_size_autoscaling.go:97
Expected error:
    <*errors.errorString | 0xc20844dcf0>: {
        s: "timeout waiting 20m0s for cluster size to be 4",
    }
    timeout waiting 20m0s for cluster size to be 4
not to have occurred
```

**Comment by piosz** [comment]
The test requires monitoring to be set to googleinfluxdb which is not the default option.

**Comment by mwielgus** [comment]
Can we either disable the test in this suit or switch to googleinfluxdb?

**Comment by piosz** [comment]
Let's disable it as for now. Historically cluster autoscaling e2e tests are running in kubernetes-e2e-gce-autoscaling suite due to possible issue with GCM quota, though we haven't seen any problem with it for at least last 2 months so we may want to revisit the decision.

@jszczepkowski @fgrzadkowski @ixdy

**Comment by gmarek** [comment]
I failed in the autoscaling suite:
http://kubekins.dls.corp.google.com/view/Critical%20Builds/job/kubernetes-e2e-gce-autoscaling/

```
/go/src/k8s.io/kubernetes/_output/dockerized/go/src/k8s.io/kubernetes/test/e2e/cluster_size_autoscaling.go:81 Expected error: <*errors.errorString | 0xc2084c9920>: { s: "timeout waiting 30m0s for cluster size to be 3", } timeout waiting 30m0s for cluster size to be 3 not to have occurred
```

**Comment by piosz** [comment]
The original issue was about monitoring configuration incompatible with autoscaler in suite kubernetes-e2e-gce-slow. We decided to not run those tests there and run them only in autoscaling suite, so it resolved the problem

According to the problem reported by @gmarek I investigated it and it seems to be a flakiness of GCE Autoscaler rather than something broken in Kubernetes itself. I'll monitor the test and create a separate issue in case of another failure.

I'm closing the issue.
